koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -730,21 +745,6 @@ def populate_monthly_cost(self, node_cost, start_date=None, end_date=None):
             end_date (datetime, str): The end_date to calculate monthly_cost.
 
         """
-        if isinstance(start_date, str):
```

Comment:
Although I'm not convinced we want exactly this date handling here, for now don't delete this block of code in populate_monthly_cost. It's a funny special case where we allow the cost of a node per month, so the start and end are always supposed to be special.